coops-ui/src/interfaces/address-interfaces.ts

```diff
@@ -0,0 +1,20 @@
+import Vue from 'vue'
+
+export interface BaseAddressType extends Vue {
+  $refs: any
+}
+
+export interface AddressIF extends Vue {
```

Comment:
Awesome, I love this!
If this should match the address schema then please add a suitable comment. (For example, in some cases, region can be null.)

Reply:
Cool, thanks!   Okie doke, sounds great, i will look into that!